src/main/java/com/minecolonies/core/network/messages/client/SyncPathMessage.java

```diff
@@ -72,6 +84,24 @@ public void toBytes(final FriendlyByteBuf buf)
         {
             node.serializeToBuf(buf);
         }
+
+        buf.writeInt(debugNodesVisitedLater.size());
```

Comment:
in 1.20.4 there is write/readCollection (not sure its in earlier 1.20)

Reply:
only some oddly typed ones that need a buffer writer